<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GromacsAnalysis</class>
 <widget class="QWidget" name="GromacsAnalysis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gromacs Analysis</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>441</width>
     <height>351</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::West</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="energy_tab">
    <attribute name="title">
     <string>Energy</string>
    </attribute>
    <widget class="QGroupBox" name="Energy_groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>371</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>Analysis</string>
     </property>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>123</width>
        <height>100</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QRadioButton" name="epot_radioButton">
         <property name="text">
          <string>Energy Potential</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="temperature_radioButton">
         <property name="text">
          <string>Temperature</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="pressure_radioButton">
         <property name="text">
          <string>Pressure</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QRadioButton" name="density_radioButton">
         <property name="text">
          <string>Density</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QPushButton" name="runEnergy_pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>75</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>RUN 
ANALYSIS</string>
     </property>
    </widget>
    <widget class="QPushButton" name="viewGraphEn_pushButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>280</y>
       <width>75</width>
       <height>51</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>VIEW 
GRAPH</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>286</width>
       <height>52</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Energy File  (*.edr)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="energyfile_lineEdit"/>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="energyfile_toolButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Output (*.xvg)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="energyout_lineEdit"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="trj_tab">
    <attribute name="title">
     <string>Trajectory</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QTextEdit" name="outputText_textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>451</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>281</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Working Directory</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="workDir_lineEdit"/>
    </item>
    <item row="0" column="2">
     <widget class="QToolButton" name="workDir_toolButton">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
